symphony/app/fbcnms-projects/hub/app/components/Main.js

```diff
@@ -78,26 +87,29 @@ function CreateServiceForm() {
   );
 }
 
-function Main() {
-  const {user, tabs, ssoEnabled} = useContext(AppContext);
-  const classes = useStyles();
-  return (
-    <div className={classes.root}>
-      <AppSideBar
-        mainItems={<NavBarItems />}
-        secondaryItems={[]}
-        projects={getProjectLinks(tabs, user)}
-        showSettings={shouldShowSettings({
-          isSuperUser: user.isSuperUser,
-          ssoEnabled,
-        })}
-        user={nullthrows(user)}
-      />
-      <AppContent>
-        <CreateServiceForm />
-      </AppContent>
-    </div>
-  );
+function Main(isWorkflow) {
```

Comment:
I'd like to make this extensible to more pages in the future. Could we call this `function Main(subApp)` and then lower down just use `Main(<WorkflowApp/>)` and `Main(<CreateServiceForm/>)`.